app/views/energy_tariffs/energy_tariffs/_energy_tariffs_summary_table.html.erb

```diff
@@ -35,7 +35,7 @@
             <%= link_to t('common.labels.edit'), edit_school_energy_tariff_path(school, energy_tariff), class: 'btn' %>
             <%= link_to t('common.labels.delete'), school_energy_tariff_path(school, energy_tariff), method: :delete, data: {confirm: t('common.confirm') }, class: 'btn btn-danger' %>
           <% end %>
-          <% if energy_tariff.tariff_holder_type == 'SiteSettings' %>
+          <% if energy_tariff.tariff_holder_type == 'SiteSettings' && site_settings_page? %>
```

Comment:
There's no if statement checking for whether the tariff holder is a SchoolGroup which means there's no "Full details", "Edit" or "Delete" button, so I can't do anything with a tariff once created.